Documentation/README_TapToPlace.md

```diff
@@ -0,0 +1,95 @@
+        - `Surface Normal Offset`: The distance between the center of the game object to place and a surface along the surface normal, if the raycast hits a surface. By default, the surface normal offset is 0 but after Start() the surface normal offset is set to the collider's extents along the z axis.
+        - `Keep Orientation Vertical`: If true, the game object to place will remain upright and in line with Vector3.up.
+        - `Rotate According to Surface`: If false, the game object to place will not change its rotation according to the surface hit.  The object will remain facing the camera while IsBeingPlaced is true.
+        - `Magnetic Surfaces`: An array of LayerMasks to execute from highest to lowest priority. First layer mask to provide a raycast hit will be used for position calculations.
+        - `Debug Enabled`: If true and in the Unity Editor, the normal of the raycast hit will be drawn in yellow. Debug enabled is useful when `RotateAccordingToSurface` is true because it draws the normal of the surface hit which visually explains why the object is set at its current orientation.
+        - `On Placing Started`: This event is triggered once when the game object to place is selected.
+        - `On Placing Stopped`: This event is triggered once when the game object to place is unselected, placed.
+
+1.  Testing Tap to Place behavior in editor
+    - Press play and hold the space bar to show an input simulation hand.
+    - Move the hand until the cube is in focus and simulate a click with the input simulation hand by clicking with the left mouse.
+        - If colliders are not present in the scene, the object will follow the `TrackedTargetType` at the defined `Default Placement Distance`.
+    - The object will follow the movement of the `TrackedTargetType` after selection. To simulate head movement in editor, press the WASD keys. Change head rotation by clicking and holding the right mouse.
+    - To stop placing the object, click again.  The object does not need to be in focus for the stop placement click. Focus is only required for the initial click that starts the placement process.
+
+    `TrackedTargetType`: Head (Default) |  `TrackedTargetType`: Controller Ray 
+    :-------------------------:|:-------------------------:
+    ![TapToPlaceInputSimulationHead](Images/Solver/TapToPlace/TapToPlaceInputSimulationHead.gif)  |  ![TapToPlaceInputSimulationControllerRay](Images/Solver/TapToPlace/TapToPlaceInputSimulationControllerRay.gif)
+
+
+## Tap to Place Code Configurability 
+
+Tap to Place object selection timing can also be controlled via `StartPlacement()` and `StopPlacement()` instead of requiring a click event. This capability is useful for writing tests and provides an alternative method to place an object in editor without using the input simulation. 
```

Comment:
```suggestion
Tap to Place object selection timing can also be controlled via `StartPlacement()` and `StopPlacement()` instead of requiring a click event. This capability is useful for writing tests and provides an alternative method to place an object in editor without using the MRTK Input system. 
```

Reply:
Added